### PR TITLE
Update configuration_hubert.py

### PR DESCRIPTION
**Pull request by majauhar**
Expanded the config documentation by adding the default configuration and some helpful details (hopefully) of:

- [x] pad_token_id
- [x] bos_token_id
- [x] eos_token_id

Related to this issue: https://github.com/huggingface/transformers/issues/33536

# What does this PR do?

Fixes # (33536)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@stevhliu

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/hubert/configuration_hubert.py [hunk 1]
```diff
@@ -139,6 +139,18 @@ class HubertConfig(PretrainedConfig):
             instance of [`HubertForSequenceClassification`].
         classifier_proj_size (`int`, *optional*, defaults to 256):
             Dimensionality of the projection before token mean-pooling for classification.
+        pad_token_id (`int`, *optional*, defaults to 0):
+            Corresponds to the "blank token" of CTC algorithm. For more info, look at this [blog](https://distill.pub/2017/ctc/)
+            Should be defined in the `tokenizer` of class [`Wav2Vec2CTCTokenizer`](https://huggingface.co/docs/transformers/en/main_classes/tokenizer) 
+            and/or the `processor` of class [`Wav2Vec2Processor`](https://huggingface.co/docs/transformers/en/model_doc/wav2vec2#transformers.Wav2Vec2Processor)
+            Could be associated with a `tokenizer` as `tokenizer.pad_token_id` or with a `processor` as `processor.tokenizer.pad_token_id`
```
Comment (stevhliu) [inline src/transformers/models/hubert/configuration_hubert.py]:
```suggestion
            Corresponds to the [blank token](https://distill.pub/2017/ctc/) of the CTC algorithm.
            Should be defined in [`Wav2Vec2CTCTokenizer`] or [`Wav2Vec2Processor`].
            Could be associated with a `tokenizer` as `tokenizer.pad_token_id` or with a `processor` as `processor.tokenizer.pad_token_id`
```

**Review by stevhliu — COMMENTED** [review]
Thanks, just one suggestion to make it less wordy 🙂

File: src/transformers/models/hubert/configuration_hubert.py (hunk 1 above)
Comment (majauhar) [reply src/transformers/models/hubert/configuration_hubert.py]: Makes sense.

**Review by stevhliu — APPROVED** [review]
Awesome, thanks! You want to open a PR for this against the main Transformers repo?

**Comment by majauhar** [comment]
> Awesome, thanks! You want to open a PR for this against the main Transformers repo?

I just realised that the merge would be into my fork. A naive question, how do I open a PR into the transformers library? What I understand as of now is that once I do a merge into my fork's main, I can open a PR to the transformer's main. Or is there a different way of doing it?

**Comment by stevhliu** [comment]
Yeah I think that should work. We've also got some detailed instructions [here](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request) for opening a PR :)

**Comment by majauhar** [comment]
> Yeah I think that should work. We've also got some detailed instructions [here](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request) for opening a PR :)

Perfect. Thank you. I would try to make the PR.

**Pull request merged**